scripts/validate_record.py

```diff
@@ -0,0 +1,290 @@
+import requests
+import json
+import os
+import sys
+from typing import Dict, List
+import utils
+from misc import from_lower_camel_case
+
+
+logger = utils.create_logging_instance("validate_record")
+
+
+class validate_record():
```

Comment:
class name should use CamelCase convention

Reply:
fixed in 2db90ac